src/main/java/com/spotinst/sdkjava/enums/AzureStatefulNodeDiskModeEnum.java

```diff
@@ -0,0 +1,48 @@
+package com.spotinst.sdkjava.enums;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public enum AzureStatefulNodeDiskModeEnum {
```

Comment:
Move this to a folder (azure.statefulNode) inside the enums folder.

Reply:
Rename to AzureStatefulNodeDiskPersistenceModeEnum

Reply:
resolved